src/systems/buoyancy_engine/BuoyancyEngine.cc

```diff
@@ -77,8 +80,32 @@ class ignition::gazebo::systems::BuoyancyEnginePrivateData
 
   /// \brief mutex for protecting bladder volume and set point.
   public: std::mutex mtx;
+
+  /// \brief get fluid density
```

Comment:
```suggestion
  /// \brief Get fluid density based on the link origin's current position.
```

Also missing the param docs